src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -35,8 +35,12 @@ import qualified Data.Graph.Typed   as Typed
 import qualified Data.IntMap.Strict as IntMap
 import qualified Data.IntSet        as IntSet
 
--- | Compute the forest of a graph's vertices in breadth first order. Complexity:
--- /O(n+m*log n)/ time and /O(n+m)/ space.
+-- | Compute the /breadth-first search/ forest of a graph, where
+--   adjacent vertices are expanded smallest to biggest with respect
+--   to their 'Ord' instance.
+-- 
+--   Let /W/ be the number of bits in a machine word. Complexity:
+--   /O((n+m)*W)/ time and /O(n)/ space.
```

Comment:
In `IntSet` docs, the factor is defined as `min(n,64)` on a 64-bit machine, so this seems a bit imprecise.

Perhaps, we could say "`W = min(n,64)` on a 64-bit machine"?

Reply:
Yeah. To me, it's awkward that `O(64)` is really `O(1)` and I'm not sure how `O(min(n,64))` makes sense? Ie, for all `n >= 64`, `min(n,64) = 64`. I'm happy to stick to the conventions from `IntSet` in any case

Reply:
If `W` might still work, maybe "word size" instead of "number of bits in a machine word" would be sufficiently better? 

Simply going with `O(log(n))` might be better anyway?

Reply:
I suggest sticking to the notation used in the standard libraries since people are already familiar with it. We could even say somewhere at the top of the module that we are following the notation from standard libraries since we are using them in our implementaion.